src/core-layers/scatterplot-layer/scatterplot-layer-vertex.glsl.js

```diff
@@ -54,10 +56,8 @@ void main(void) {
   // 0 - solid circle, 1 - stroke with lineWidth=0
   innerUnitRadius = outline * (1.0 - strokeWidth / outerRadiusPixels);
 
-  // Find the center of the point and add the current vertex
-  vec3 center = project_position(instancePositions);
-  vec3 vertex = positions * outerRadiusPixels;
-  gl_Position = project_to_clipspace(vec4(center + vertex, 1.0));
+  vec3 offset = positions * outerRadiusPixels;
+  gl_Position = project_position_to_clipspace(instancePositions, instancePositions64xyLow, offset);
```

Comment:
Thought: Starting to wonder if it should be called `project_position64_to...` since it does take a 64 bit position, even though it is not unique to `project64`